From 33dd6c51ad7619cc1eccc8cd06e6acc63599618f Mon Sep 17 00:00:00 2001
From: jakaja <24541264+jakaja@users.noreply.github.com>
Date: Mon, 7 Oct 2019 18:56:21 +0300
Subject: [PATCH] Add rating mark also to "Shift" view

"InfoWall", "Wall" and "Poster" views already have "ratings bubble", but for some reason it was missing from "Shift" view. Adding it.
---
 addons/skin.estuary/xml/View_53_Shift.xml | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/addons/skin.estuary/xml/View_53_Shift.xml b/addons/skin.estuary/xml/View_53_Shift.xml
index d7f3d441ba41..c537f320e56f 100644
--- a/addons/skin.estuary/xml/View_53_Shift.xml
+++ b/addons/skin.estuary/xml/View_53_Shift.xml
@@ -89,6 +89,11 @@
 							<height>32</height>
 							<texture>$VAR[WallWatchedIconVar]</texture>
 						</control>
+						<control type="group">
+							<left>158</left>
+							<top>92</top>
+							<include condition="Skin.HasSetting(circle_rating) | Skin.HasSetting(circle_userrating)">RatingCircle</include>
+						</control>
 					</itemlayout>
 					<focusedlayout width="370">
 						<control type="image">
@@ -119,6 +124,11 @@
 							<height>32</height>
 							<texture>$VAR[WallWatchedIconVar]</texture>
 						</control>
+						<control type="group">
+							<left>158</left>
+							<top>92</top>
+							<include condition="Skin.HasSetting(circle_rating) | Skin.HasSetting(circle_userrating)">RatingCircle</include>
+						</control>
 					</focusedlayout>
 				</control>
 			</control>
